samples/04_gis_analysts_data_scientists/forecasting_air_temperature_in_california.ipynb

```diff
@@ -0,0 +1,2703 @@
+{
```

Comment:
<p>Here in the original dataset the <code>variable predict</code> column of Average Temperature (<code>TAVG</code>) is allocated with NaNs, for the forecasting period of 2014-2015. This format is required for the&nbsp;model.predict()&nbsp;function in time series analysis, which will fill up the NaN values with forecasted temperatures.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1119/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>